test/integrations/destinations/hs/processor/data.ts

```diff
@@ -5269,4 +5269,101 @@ export const data = [
       },
     },
   },
+  {
+    name: 'hs',
+    description: 'Test coversion of null to string values',
+    feature: 'processor',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
```

Comment:
Use payload builder for generating this paylaod check `testUtils.ts`

Reply:
done , in output couldn't use function as there are extra parameters for HS